<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FrontendNewVault</class>
 <widget class="QWidget" name="FrontendNewVault">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
	font: 100 11pt &quot;Noto Sans&quot;;
	color: rgb(255, 255, 255);
	background-color: rgb(30, 30, 30);
}
QWidget:disabled{
	color: rgb(100, 100, 100);
}

/* ToolTip */
QToolTip {
	color: rgb(255, 255, 255); 
	background: rgb(55, 55, 55);
	border: 1px solid rgb(70, 70, 70); 
	border-radius: 5px;
}

/* ProgressBar */
QProgressBar{
 	border: 1px solid rgb(55, 55, 55);
	border-radius: 5px;
	padding-left: 0px;
	color: rgb(0, 65, 74);
	text-align: center;
}
QProgressBar::chunk{
	border-radius: 5px;
 	background-color: rgb(44, 222, 133);
}

#VaultAditionalInfoLabel , #DirectoryAditionalInfoLabel, #PasswordAditionalInfoLabel{
	font: 9pt;
	color: rgb(200, 200, 200);
    min-height: 16px;
    max-height: 16px;
}
/* Label */
QLabel{
border: 0px;
}

/* TextEdit */
QTextEdit{
font: 9pt;
border: 1px solid rgb(55, 55, 55);
border-radius: 5px;
}
/* LineEdit */
QLineEdit{
	border: 1px solid rgb(55, 55, 55);
	border-radius: 5px;
}

/* ListView */
QListView{
	border: 1px solid rgb(55, 55, 55);
	border-radius: 5px;
	font: 11pt;
	gridline-color: rgb(0, 0, 0);
}

/* ComboBox */
QComboBox{
	background-color: rgb(40, 40, 40);
	border: 1px solid rgb(55, 55, 55);
	border-radius: 5px;
	padding-left: 3px;
}
QComboBox::drop-down {
    width: 20px;
    border-left: 1px solid rgb(55, 55, 55);
    border-top-right-radius: 5px; 
    border-bottom-right-radius: 5px;
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;
}
QComboBox::down-arrow {
    image: url(:/icons/arrow_down.png);
}

/* PushButton */
QPushButton{
	border: 1px solid rgb(55, 55, 55);
	border-radius: 5px;
	background-color: rgb(40, 40, 40);
}
QPushButton:pressed{
	background-color: rgb(30, 30, 30);
}

/* CheckBox */
QCheckBox{
	border: 1px solid rgb(55, 55, 55);
	border-radius: 5px;
}
QCheckBox::indicator{
	margin-left: 2px;
	margin-right: -2px;
}
QCheckBox::indicator:unchecked{
	border: 1px solid rgb(0, 0, 0);
	background-color: rgb(200, 40 ,40);
}
QCheckBox::indicator:checked{
	border: 1px solid rgb(0, 0, 0);
	background-color: rgb(40, 200 ,40);
}

</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="6" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="PwConfirmLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Confirm Password</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="PwConfirmLineedit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PwConfirmVisibilityButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../asset/resource.qrc">
         <normaloff>:/icons/visibility_off.png</normaloff>
         <normalon>:/icons/visibility_on.png</normalon>:/icons/visibility_off.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="CreateVaultButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Create</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="1">
      <widget class="QLineEdit" name="PasswordLineedit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::EchoMode::Password</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="PasswordVisibilityButton">
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../asset/resource.qrc">
         <normaloff>:/icons/visibility_off.png</normaloff>
         <normalon>:/icons/visibility_on.png</normalon>:/icons/visibility_off.png</iconset>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="PasswordLabel">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QLabel" name="PasswordAditionalInfoLabel">
       <property name="text">
        <string>Password must be 4-32 characters, and can include alphabets, numbers, symbols</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="ModeLabel">
     <property name="text">
      <string>Mode</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QLineEdit" name="VaultNameLineedit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QLabel" name="VaultAditionalInfoLabel">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="VaultLabel">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Vault</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="DirectoryLabel">
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>130</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string>Folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="DirectoryPathLabel">
       <property name="styleSheet">
        <string notr="true">QWidget{
	padding-left: 1px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="DirectoryOpenButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../asset/resource.qrc">
         <normaloff>:/icons/folder_open.png</normaloff>:/icons/folder_open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../asset/resource.qrc"/>
 </resources>
 <connections/>
</ui>
